Source/Core/SelectableSection.swift

```diff
@@ -132,7 +132,12 @@ open class SelectableSection<Row: SelectableRowType> : Section, SelectableSectio
         self.selectionType = selectionType
         super.init(header, initializer)
     }
-    
+
+    public init(header: String, footer: String, selectionType: SelectionType, _ initializer: (Section) -> () = { _ in }) {
```

Comment:
It will be still possible to use custom headers by setting them manually as section property, like described in the link that you shared. We are doing exactly that in our app. 
Currently `Section` initialisers are not providing `HeaderFooterViewRepresentable` as parameter, only `String`. It will make sense to improve that but I believe it should be done in a separate PR.